### ISSUE TITLE
Animation is very slow until i change tabs

### ISSUE THREAD

**Issue by FaryarKankash** [maxeth/react-type-animation#55]
`
                            <TypeAnimation
                                sequence={[
                                    `> question: ${item.question}`,
                                ]}
                                wrapper="p"
                                speed={{ type: "keyStrokeDelayInMs", value: 0.001 }}
                                className='font-secondary'
                            />
`

**Comment by maxeth** [comment]
I just tested it and in my case it's very fast, not slow, just as it should with a delay of 0.001ms between keystrokes. Not sure why it's slow on your end. Maybe this is related? https://github.com/maxeth/react-type-animation/issues/53

**Comment by FaryarKankash** [comment]
> I just tested it and in my case it's very fast, not slow, just as it should with a delay of 0.001ms between keystrokes. Not sure why it's slow on your end. Maybe this is related? #53

do you test it in next js? i have this problem in next js app router client side

**Comment by malmahdawi** [comment]
@FaryarKankash yes i faced this while using nextjs and passing number. use this speed prop instead `speed={{type: "keyStrokeDelayInMs", value: 20}}`
